plugins/pdns/pdns_rel

```diff
@@ -31,17 +36,19 @@ if [ "$1" = "config" ]; then
         exit 0
 fi
 
-hits=$($command packetcache-hit | awk -F= '{print $2}')
-queries=$($command udp-queries | awk -F= '{print $2}')
-old_hits=$(cat $state_file | head -n1)
-old_queries=$(cat $state_file | tail -n1)
+hits=$($command packetcache-hit)
+queries=$($command udp-queries)
+if [ -f $state_file ]; then
+        old_hits=$(cat $state_file | head -n1)
+        old_queries=$(cat $state_file | tail -n1)
+fi
 
 if [ -f $state_file ] && [ $(ls -l --time-style=+%s $state_file | awk '{print $6}') -gt $(date --date="7 minutes ago" +%s) ] ; then
```

Comment:
This is not your piece of the code, but maybe you are in the mood ...
`date --reference %s` is probably a much safer choice than manual parsing of `ls`.
Additionally quoting around all variables and command substitutions would be good.

Reply:
Quoting the two commands is not needed because the output is a single number not a string.